### PR TITLE
store: log detailed region count when start.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing @hhkbp2

### CODE REVIEW

**Comment by disksing** [comment]
LGTM

File: src/raftstore/store/store.rs [hunk 1]
```diff
@@ -171,6 +179,12 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             Ok(true)
         }));
 
+        info!("[store {}] starts with {} regions, including {} tombstone and {} applying ones",
```
Comment (queenypingcap) [inline src/raftstore/store/store.rs]: +        info!("[store {}] starts with {} regions, including {} tombstones and {} applying regions",

File: src/raftstore/store/store.rs [hunk 2]
```diff
@@ -171,6 +179,12 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             Ok(true)
         }));
 
+        info!("[store {}] starts with {} regions, including {} tombstones and {} applying regions",
```
Comment (ngaut) [inline src/raftstore/store/store.rs]: We should also log the time we used when scan all the region meta.

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**